docs/user_docs.ipynb

```diff
@@ -796,13 +799,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sources = [scarlet.ExtendedSource((src['y'],src['x']), scene, observation, bg_rms) for src in catalog]\n",
-    "blend = scarlet.Blend(scene, sources, observation)\n",
-    "blend.fit(20)\n",
+    "blend.fit(10)\n",
     "print(blend.it) \n",
-    "blend.fit(20)\n",
+    "blend.fit(10)\n",
     "print(blend.it)\n",
-    "blend.fit(20)\n",
+    "blend.fit(10)\n",
```

Comment:
Good catch. You should also change the text that follows from "...60 iterations" to "...30 iterations".